docs/user_guide/testing.md

```diff
@@ -0,0 +1,25 @@
+# tRPC-Go Testing
```

Comment:
Could please also update the corresponding Chinese version? Just using chatGPT to generate should be fine.

Reply:
Sure.